packages/react/src/utils/css-in-js.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  emotionCoreVersion,
+  emotionStyledVersion,
+  styledComponentVersion
+} from './versions';
+import { PackageDependencies } from './dependencies';
+
+export function reduceCssInJsModules(
+  extraPackageDependencies: null | PackageDependencies,
+  style: string
+): null | PackageDependencies {
+  const deps = extraPackageDependencies
+    ? extraPackageDependencies.dependencies
+    : {};
+  const devDeps = extraPackageDependencies
+    ? extraPackageDependencies.devDependencies
+    : {};
+  switch (style) {
+    case 'styled-components':
+      return {
+        dependencies: {
+          ...deps,
+          'styled-components': styledComponentVersion
+        },
+        devDependencies: { ...devDeps }
+      };
+    case 'emotion':
+      return {
+        dependencies: {
+          ...deps,
+          '@emotion/styled': emotionStyledVersion,
+          '@emotion/core': emotionCoreVersion
+        },
+        devDependencies: { ...devDeps }
+      };
+    default:
+      return extraPackageDependencies;
+  }
+}
+
+export function getCssInJsModule(style: string): null | string {
```

Comment:
I think it's easier to have the if else in the file itself.. so you don't have to go dig for where the function is.

Reply:
It's used by app and component schematics.. but I've changed this in the updated code anyway.